From 57e0e583f7fca06092eb64d9f70a0e2226708563 Mon Sep 17 00:00:00 2001
From: Johnny Robeson <johnny@localmomentum.net>
Date: Thu, 15 Jan 2015 13:30:37 -0800
Subject: [PATCH] Remove $ENV config from FindMySQL.cmake

Summary: These are seemingly superflous, considering that you can
still configure which library and include directory to use
with cmake -D parameters.

I was figuring out why

 # cmake .

 show showing me:

  -- MySQL Include dir: /usr/include/mysql  library dir: /lib/mysql

when it should have been:

  -- MySQL Include dir: /usr/include/mysql  library dir: /usr/lib64/mysqlh

For some reason that i don't comprehend, this particular line was causing the problem:

$ENV{MYSQL_DIR}/lib/mysql

I didn't touch the windows section because such variables may or may not still be useful there, I don't know enough about windows stuff to be able to tell.

If you think they should be removed too, then i will add that to this PR also.
Closes https://github.com/facebook/hhvm/pull/4648

Reviewed By: @JoelMarcey

Differential Revision: D1781763

Signature: t1:1781763:1421357350:9b8239e9dea848bf99f14bd596bf9d53421c3685
---
 CMake/FindMySQL.cmake | 5 -----
 1 file changed, 5 deletions(-)

diff --git a/CMake/FindMySQL.cmake b/CMake/FindMySQL.cmake
index d27d294..b047094 100644
--- a/CMake/FindMySQL.cmake
+++ b/CMake/FindMySQL.cmake
@@ -31,8 +31,6 @@
 
 #-------------- FIND MYSQL_INCLUDE_DIR ------------------
 FIND_PATH(MYSQL_INCLUDE_DIR mysql.h
-  $ENV{MYSQL_INCLUDE_DIR}
-  $ENV{MYSQL_DIR}/include
   /usr/include/mysql
   /usr/local/include/mysql
   /opt/mysql/mysql/include
@@ -70,9 +68,6 @@ IF (WIN32)
 ELSE (WIN32)
   FIND_LIBRARY(MYSQL_LIB NAMES mysqlclient_r
     PATHS
-    $ENV{MYSQL_DIR}/libmysql_r/.libs
-    $ENV{MYSQL_DIR}/lib
-    $ENV{MYSQL_DIR}/lib/mysql
     /usr/lib/mysql
     /usr/local/lib/mysql
     /usr/local/mysql/lib
